### PR TITLE
Translate OSPO definition into Japanese

### PR DESCRIPTION
**Pull request by shu-mutou**
Create README-ja.md by translating README.md

### CODE REVIEW

**Comment by shu-mutou** [comment]
Please let me know if I'm going about it the wrong way. Thanks in advance.

File: README-ja.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+### ビジネス指向の細分化
+
+リスクの軽減、エンジニアの手法の改善、経済的利益の実現（ニーモニック：恐怖、愛、そしてお金）
+
+**法的リスクの軽減：** 多くの場合、企業が最初に懸念するのは法令順守に関するものです。OSPO は多くの場合、企業のオープンソースライセンスコンプライアンスプロセスの側面を監督します。通常、ソフトウェアを配布する企業はこれを最も懸念しており、法的リスクの回避を中心に OSPO を開始します。
+この分野におけるプログラムオフィスの責任には次のものが含まれます。
+* オープンソースライセンスのコンプライアンスのレビューと監視の維持
+* 導入されるコードの使用に関するレビュープロセスの実行
+* 企業がオープンソースプロジェクトに効果的に貢献できるようにすること
```
Comment (ziyi-xie) [inline README-ja.md:71]:
```suggestion
* 企業がオープンソースプロジェクトに効果的に貢献するようにすること
```

Comment (shu-mutou) [reply README-ja.md:71]: 「するようにすること」はちょっとおかしいので、そのままにします。

**Comment by shu-mutou** [comment]
Thank you for reviewing @ziyi-xie !! :heart:

**Comment by node** [comment]
The better choice : 
- support i18n on the whole glossary repos  https://github.com/todogroup/glossary
- translate OSPO definition and others into more languages 

😄

**Comment by shu-mutou** [comment]
I'd like to translate whole of grossary into Japanese in the future. :wink:

**Comment by ziyi-xie** [comment]
Thanks! LGTM

**Review by anajsana — APPROVED** [review]

**Comment by anajsana** [comment]
> I'd like to translate whole of grossary into Japanese in the future. 😉

Awesome! There is a dedicated section for contributors dedicated to contribute with translations in the contribution ladder and how to become a `localization approver`. https://ospoglossary.todogroup.org/contributor-ladder/#localization-approvers

**Pull request merged**